core/src/main/java/hudson/model/AbstractProject.java

```diff
@@ -1153,10 +1143,11 @@ public String getShortDescription() {
         }
     }
 
-    public CauseOfBlockage getCauseOfBlockage() {
+    @Override public CauseOfBlockage getCauseOfBlockage() {
```

Comment:
When you will debug 5-7y old code you will hate coding style mess and not squashed commits ;)